apps/v3_api/lib/routes.ex

```diff
@@ -11,7 +11,7 @@ defmodule V3Api.Routes do
   end
 
   def get(id, opts \\ []) do
-    get_json("/routes/#{id}", [], opts)
+    get_json("/routes/#{id}", opts)
```

Comment:
@digitalcora whither removing that one `[]`?

Reply:
It's my belief this was either a mistake or an artifact of some removed code. Other functions in this module pass their params as the second argument to `get_json` — the third argument is "opts" and allows overriding things like the API key and request timeout. In master there is no code that calls `get` with a second argument, so it's a mystery why this was here. However, we now actually need to pass some request params, so I aligned this with the other functions.